docs/source/releases/v1_12_2a0.rst

```diff
@@ -13,6 +13,27 @@
 Version 1.12.2a0 (2024-02-21)
 *****************************
 
+* Documentation
+
+  * Added documentation for create_plugin_registries and PluginRegistry Class
+
+Documentation
+=============
+
+Added Documentation for create_plugin_registries and PluginRegistry Class
+-------------------------------------------------------------------------
+
+With the addition of create_plugin_registries and the new PluginRegistry Class, we have
+a need to add documentation describing what these do, how they are used, and the
+benefits that come from adding them to the main GeoIPS code. This branch does exactly
+that, and adds a section under 'UserGuide' describing what these pieces of code actually
+do.
+
+::
+
+    modified: geoips/docs/source/userguide/index.rst
+    added: geops/docs/source/userguide/plugin_registries.rst
+=======
```

Comment:
This should be removed also - from a merge conflict.

Reply:
This appears to be fixed.